ocis-reva/CHANGELOG.md

```diff
@@ -0,0 +1,948 @@
+
+   - Update reva to v1.2.1-0.20200826162318-c0f54e1f37ea - Do not swallow 'not found' errors in
+   Stat [(reva/#1124)](https://github.com/cs3org/reva/pull/1124) - Rewire dav files to the
+   home storage [(reva/#1125)](https://github.com/cs3org/reva/pull/1125) - Do not restore
+   recycle entry on purge [(reva/#1099)](https://github.com/cs3org/reva/pull/1099) -
+   Allow listing the trashbin [(reva/#1091)](https://github.com/cs3org/reva/pull/1091) -
+   Restore and delete trash items via ocs
+   [(reva/#1103)](https://github.com/cs3org/reva/pull/1103) - Ensure ignoring public
+   stray shares [(reva/#1090)](https://github.com/cs3org/reva/pull/1090) - Ensure
+   ignoring stray shares [(reva/#1064)](https://github.com/cs3org/reva/pull/1064) -
+   Minor fixes in reva cmd, gateway uploads and smtpclient
+   [(reva/#1082)](https://github.com/cs3org/reva/pull/1082) - Owncloud driver -
+   propagate mtime on RemoveGrant
+   [(reva/#1115)](https://github.com/cs3org/reva/pull/1115) - Handle redirection
+   prefixes when extracting destination from URL
+   [(reva/#1111)](https://github.com/cs3org/reva/pull/1111) - Add UID and GID in ldap auth
+   driver [(reva/#1101)](https://github.com/cs3org/reva/pull/1101) - Add calens check to
+   verify changelog entries in CI
+   [(reva/#1077)](https://github.com/cs3org/reva/pull/1077) - Refactor Reva CLI with
+   prompts [(reva/#1072)](https://github.com/cs3org/reva/pull/1072j) - Get file info
+   using fxids from EOS [(reva/#1079)](https://github.com/cs3org/reva/pull/1079) - Update
+   LDAP user driver [(reva/#1088)](https://github.com/cs3org/reva/pull/1088) - System
+   information metrics cleanup
+   [(reva/#1114)](https://github.com/cs3org/reva/pull/1114) - System information
+   included in Prometheus metrics
+   [(reva/#1071)](https://github.com/cs3org/reva/pull/1071) - Add logic for resolving
+   storage references over webdav
+   [(reva/#1094)](https://github.com/cs3org/reva/pull/1094)
+
+   https://github.com/owncloud/ocis/ocis-revapull/454
+
+
+* Enhancement - Update reva to v1.2.1-0.20200911111727-51649e37df2d: [#466](https://github.com/owncloud/ocis/ocis-revapull/466)
+
+   - Update reva to v1.2.1-0.20200911111727-51649e37df2d - Added new OCIS storage driver ocis
+   [(reva/#1155)](https://github.com/cs3org/reva/pull/1155) - App provider: fallback to
+   env. variable if 'iopsecret' unset
+   [(reva/#1146)](https://github.com/cs3org/reva/pull/1146) - Add switch to database
+   [(reva/#1135)](https://github.com/cs3org/reva/pull/1135) - Add the ocdav HTTP svc to the
+   standalone config [(reva/#1128)](https://github.com/cs3org/reva/pull/1128)
+
+   https://github.com/owncloud/ocis/ocis-revapull/466
+
+# Changelog for [0.13.0] (2020-08-27)
+
+The following sections list the changes in ocis-reva 0.13.0.
+
+[0.13.0]: https://github.com/owncloud/ocis/ocis-revacompare/v0.12.0...v0.13.0
```

Comment:
Codacy found an issue: [[final-definition] Move definitions to the end of the file (after the node at line `942`)](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)